SKILL.tsv

```diff
@@ -1655,17 +1655,17 @@ SKILL_20150317_001654	Calm down and stop moving.
 SKILL_20150317_001655	Addiction
 SKILL_20150317_001656	Attack is increased while defense is reduced.
 SKILL_20150317_001657	Transparent
-SKILL_20150317_001658	Transparent state.
+SKILL_20150317_001658	$Transparent Status Effect.
 SKILL_20150317_001659	Recover HP every 3 seconds.
 SKILL_20150317_001660	monster Hold
 SKILL_20150317_001661	Monster hold script link
-SKILL_20150317_001662	Attack increase
+SKILL_20150317_001662	$Increased Attack
 SKILL_20150317_001663	Attack and evasion decrease
-SKILL_20150317_001664	Increase SP, Stamina
-SKILL_20150317_001665	Amount of SP,stamina recovery increase
-SKILL_20150317_001666	Defense and attack will be reduced.
-SKILL_20150317_001667	Rage
-SKILL_20150317_001668	Become enraged. Intelligence, physical defense, and magic defense are reduced while strength, physical attack, and movement speed are increased.
+SKILL_20150317_001664	$Increased SP & Stamina
+SKILL_20150317_001665	$Increases the amount of SP and Stamina recovery
+SKILL_20150317_001666	$Decreased Defense and Attack.
+SKILL_20150317_001667	$Rage
+SKILL_20150317_001668	$Become enraged. Decreases INT, Physical and Magic Defense, but increases STR, Physical Attack and Movement Speed.
```

Comment:
I'd rearrange this to "Increases STR, Physical Attack and Movement Speed but decreases INT, Physical and Magic Defense." since the player would rather it's been customary to state the benefits of rage mode before the drawbacks in games.